tests/fixtures/saas/zendesk_fixtures.py

```diff
@@ -88,3 +100,66 @@ def zendesk_dataset_config(
     )
     yield dataset
     dataset.delete(db=db)
+
+
+@pytest.fixture(scope="function")
+def zendesk_create_erasure_data(
+    zendesk_connection_config: ConnectionConfig, zendesk_erasure_identity_email: str
+) -> None:
+
+    zendesk_secrets = zendesk_connection_config.secrets
+    base_url = f"https://{zendesk_secrets['domain']}"
+    headers = {
+        "Content-Type": "application/json",
+        "Authorization": "Basic {}".format(
+            base64.b64encode(
+                bytes(
+                    f"{zendesk_secrets['username']}:{zendesk_secrets['api_key']}",
+                    "utf-8",
+                )
+            ).decode("ascii")
+        ),
+    }
+
+    connector = SaaSConnector(zendesk_connection_config)
```

Comment:
Can you rewrite this call to use `requests` instead of creating a `SaaSConnector`. I'd like to stay away from using SaaS connector logic to create test data.